halo/app/monitor.go

```diff
@@ -78,12 +83,12 @@ func monitorCometOnce(ctx context.Context, rpcClient rpcclient.Client, isSyncing
 
 // monitorEVMForever blocks until the contract is canceled.
 // It periodically calls monitorEVMOnce.
-func monitorEVMForever(ctx context.Context, cfg Config, ethCl ethclient.Client) {
+func monitorEVMForever(ctx context.Context, cfg Config, ethCl ethclient.Client, readiness *ReadyResponse) {
 	if cfg.Network == netconf.Simnet {
 		return // Simnet doesn't have an EVM tp monitor.
 	}
 
-	ticker := time.NewTicker(time.Second * 30)
+	ticker := time.NewTicker(time.Second * 10)
 	defer ticker.Stop()
 
 	// Geth Auth API (EngineClient) doesn't enable net module, so we can't monitor peer count with it.
```

Comment:
see here. Most halo instances only have engine API client, so cannot monitor geth peers. In this case, skip gethpeers==0 checks.